### ISSUE TITLE
Engine BE 2: Update scheduler to also run  Engine Sidekiq jobs

### ISSUE THREAD

**Issue by kayline** [department-of-veterans-affairs/va.gov-team#98352]
## Value Statement

**_As a_** backend dev

**_I want to_** to be sure the new Sidekiq jobs work

**_So that_** so that we can later turn off the old versions

---

## Background Context

- Part of the Engine epic
- Only add the jobs that can re-run safely (NOT form4142_submit and submit_upload, which are not scheduled)

## Tasks

- [ ] Ensure logging is different for the engine and non-engine job classes
- [ ] Set engine jobs to run before current jobs
- [ ] Monitor for at least a week to ensure correct behavior

## Dependencies 

- Add here

## Acceptance Criteria

- [ ] _AC_
- [ ] _AC_
- [ ] _AC_

## Out of scope

- Add here

## Open questions

- Add here

## Designs and Build Notes

- Add here

## Outcome, Success Measure, KPI(S), and Tracking Link

- Add here

### Enablement team (if needed)
@ member of team

## Definition of Ready

- [ ] Clear value description
- [ ] Testable acceptance criteria
- [ ] Accessibility added to acceptance criteria
- [ ] Approved designs attached
- [ ] Sample data provided where appropriate
- [ ] Estimated to fit within the sprint
- [ ] Dependencies and blockers linked

## Definition of Done

- [ ] Meets acceptance criteria
- [ ] Passed E2E testing (90% coverage)
- [ ] Passed unit testing (90% coverage)
- [ ] Passed integration testing (if applicable)
- [ ] Code reviewed (internal)
- [ ] Submitted to staging
- [ ] Team approved production verification process
- [ ] Reviewed and approved by product and/or design

**Comment by kayline** [comment]
Draft PR ready, waiting on Step 1 to get merged

**Comment by kayline** [comment]
Merged, needs staging testing and then keep an eye on jobs in prod
